cmd/kube-proxy/app/server.go

```diff
@@ -230,7 +232,6 @@ func NewOptions() *Options {
 // Complete completes all the required options.
 func (o *Options) Complete() error {
 	if len(o.ConfigFile) == 0 && len(o.WriteConfigTo) == 0 {
-		klog.InfoS("Warning, all flags other than --config, --write-config-to, and --cleanup are deprecated, please begin using a config file ASAP")
```

Comment:
separate commit/PR seems fine, but we shouldn't be warning people to stop using flags if the config file is still alpha level

Reply:
@liggitt  we have this situation for years in kube-proxy and we are not proud of it, but the reality is that there is no better alternative than use the config, the level is just because nobody spend time in graduating it

Reply:
> no better alternative than use the config

I mean... they could use the flags we're warning them to stop using :)

Reply:
we had issues because config override flags and people get confused

Reply:
warning in that case seems fine; here we're warning if they *didn't* specify a config file

Reply:
So we go back to this warning, and I open an issue for tracking this. see https://github.com/kubernetes/kubernetes/issues/118090

Reply:
But if we go back to this warning,  the  log warning runs before ValidateAndApplyAsField, Is it acceptable?